### DISCUSSION TITLE
Change date input for #_BIRTHDATE

### DISCUSSION THREAD

**Discussion by MichaelWybo** [liedekef/events-made-easy#304] [General]
Hello Franky, 

Now the "#_BIRTHDATE" placeholder use the DATTE (JS) as input, is it possible to change it as DATE (HTML5) Type as we can do in the template ?

Thanks

**Comment by liedekef** [comment]
Not for the birthday itself, no. But you're free to create a custom field of type "Date (HTML5)" with purpose "People field" and use that as birthday in your forms.
